scripts/o3de/o3de/repo_properties.py

```diff
@@ -166,7 +178,9 @@ def _edit_objects(object_typename:str,
                                 archive_filename, 
                                 release_archive_path, 
                                 repo_json['repo_uri'], 
+                                force,
                                 download_prefix)
+                    if type(json_data) == int: return 1
```

Comment:
add a comment about what would cause json_data to be an int.
can also change this to be `if type(json_data) != Dict:`